### DISCUSSION TITLE
How to clear some circle in PIXI.Graphics()?

### DISCUSSION THREAD

**Discussion by jingyuLin1999** [pixijs/pixijs#8671] [Q&A]
I draw two circle, and want to clear first circle, how can i do it.
```js
let circle = new PIXI.Graphics();
stage.addChild(circle);
circle.beginFill(0xff0000);
circle.drawCircle(0, 0, 10);

circle.beginFill(0xff0000);
circle.drawCircle(20, 20, 10);
circle.endFill();
```
In my project, I have drawn 200000 circles and want to remove the first 100 circles.
not clear all and redraw (200000  - 100) circles.

**Comment by domis86** [comment]
draw each circle as separate Graphics object, and then remove objects which you dont need.

```
let circle1 = new PIXI.Graphics();
stage.addChild(circle1);
circle1.beginFill(0xff0000);
circle1.drawCircle(0, 0, 10);

let circle2 = new PIXI.Graphics();
circle2.beginFill(0xff0000);
circle2.drawCircle(20, 20, 10);
circle2.endFill();
```

(of course use some array etc of objects instead variable per each ;) )

**Comment by jingyuLin1999** [comment]
Thanks for your reply. this maybe can resolve my issue.  I find use your method will use less memory. thanks again!

**Comment by jingyuLin1999** [comment]
when i draw line by your method, it will take more memory.  when i draw circle will use less memory. why?
